pkg/util/pidlimit_test.go

```diff
@@ -17,12 +17,18 @@ limitations under the License.
 package util
 
 import (
+	"os"
 	"testing"
 )
 
 // minimal test to check if GetPIDLimit() returns an int
 // changing the limit require root permissions, not tested
-func TestGetPIDLimix(t *testing.T) {
+func TestGetPIDLimit(t *testing.T) {
+	runTest := os.Getenv("CEPH_CSI_RUN_ALL_TESTS")
```

Comment:
@nixpanic sorry I forgot about it in the last review, can we update the doc for this ENV?

Reply:
Added a note about it in the developers guide.